lib/matplotlib/patches.pyi

```diff
@@ -455,8 +455,10 @@ class BoxStyle(_Style):
 
 class ConnectionStyle(_Style):
     class _Base(ConnectionStyle):
+        @ api.deprecated("3.7")
         class SimpleEvent:
-            def __init__(self, xy: tuple[float, float]) -> None: ...
+            def __init__(self, xy):
+                self.x, self.y = xy
```

Comment:
We do not wish to add this back, especially as this is a `pyi` stub file (where `@api.deprecated` has no effect)